example/spec/helpers/apiUtils.js

```diff
@@ -35,8 +35,7 @@ async function waitForModelStatus(model, params, status) {
       }
 
       return record;
-    },
-    { retries: 10 }
+    }
```

Comment:
We don't want to cap `waitForModelStatus` at all? Seems like capping it at 10 retries is pretty reasonable? Otherwise tests could hang indefinitely?

Reply:
From the `retry` docs: `retries: The maximum amount of times to retry the operation. Default is 10. Seting this to 1 means do it once, then retry it once.` As default is 10, this line was completely redundant.